specification/azsadmin/resource-manager/compute/readme.md

```diff
@@ -37,6 +37,8 @@ input-file:
     - Microsoft.Compute.Admin/preview/2015-12-01-preview/PlatformImages.json
     - Microsoft.Compute.Admin/preview/2015-12-01-preview/Quotas.json
     - Microsoft.Compute.Admin/preview/2015-12-01-preview/VMExtensions.json
+    - Microsoft.Compute.Admin/preview/2018-07-30-preview/Disks.json
+    - Microsoft.Compute.Admin/preview/2018-07-30-preview/DiskMigrations.json
```

Comment:
These new APIs were only added to package-2015-12-01-preview, is that intentional or should they be added to more packages?

Reply:
I am not sure what's does the package mean here.  We add new resource with new api version. For existing resources, they keep the old api version. But all those resources should be shipped within one SDK module and one ps module. 

What should I do? thanks

---
In reply to: [202755792](https://github.com/Azure/azure-rest-api-specs/pull/3418#discussion_r202755792) [](ancestors = 202755792)

Reply:
Oh my mistake, there is only one package for this service so no need to do anything further here.